Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6215,6 +6215,36 @@ def get_anti_spyware_best_practice_command():
     })
 
 
+def apply_dns_signature_policy_command(args) -> CommandResults:
```

Comment:
please add a doc-string to the function.

Reply:
I will add it after the doc-review by Shirly :)